python/surf/devices/nxp/_Sa56004x.py

```diff
@@ -559,4 +559,13 @@ def addBoolPair(name,description,rdOffset,wrOffset,bitOffset):
         ))
         
         ############################################################################
-        
\ No newline at end of file
+        
+        # Default to simple view
+        self.simpleView()        
+        
+    def simpleView(self):
```

Comment:
You'll probably want to keep the "enabled" Variable unhidden as well.

Reply:
This is a copy of the Xadc procedure

Reply:
Fixed in adcf872